lib/ae_mdw_web/websocket/broadcaster.ex

```diff
@@ -172,7 +160,9 @@ defmodule AeMdwWeb.Websocket.Broadcaster do
   end
 
   defp broadcast(channel, msg) do
-    AeMdwWeb.Websocket.SocketHandler.channel_broadcast(channel, msg)
```

Comment:
wasn't that way simpler before in only relaying the message once, or was the other action "implicit"?

Reply:
If you look to the Riverside code it relays it once because it has the concept of channels. From the previous commit after removing riverside it needs to relay the message multiple times (Phoenix websocket is just a transport):
https://github.com/aeternity/ae_mdw/blob/4b69441dfeb5425ec63bef8e63d1c44dfa90fd42/lib/ae_mdw_web/websocket/socket_handler.ex#L37-L41

Clarifying a bit more, the difference here is that all ets/channel code that was scattered throughout 3 modules were delegated to a specific module. The `AeMdwWeb.Websocket.Subscriptions` was created for this otherwise the `SocketHandler` would continue having protocol handling code mixed with subscription.

This way the semantics of latest `broadcast` would be:
```
    channel 
    |> Subscriptions.subscribers() # for all subscribers of this channel
    |> Enum.each(&SocketHandler.send(&1, msg)) # send for each one of them
```

Reply:
thanks for the clarification